src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,13 @@
+import classes from "./Icon.module.scss";
+
+interface IIcon {
+  iconType: string;
+}
+
+const Icon = ({iconType} : IIcon) => {
```

Comment:
I think it should have different name since it's not classic icon and it's being used in one place only